security/pkg/k8s/controller/monitoring.go

```diff
@@ -26,27 +26,27 @@ var (
 	errorTag = monitoring.MustCreateLabel(errorlabel)
 
 	serviceAccountCreationCounts = monitoring.NewSum(
-		"svc_acc_created_cert_count",
+		"citadel_svc_acc_created_cert_count",
```

Comment:
There is one metric test here, you may want to update it. https://github.com/istio/istio/blob/ceba93605ab0ef34237ce294d1141ceecd09a0fc/tests/e2e/tests/dashboard/dashboard_test.go#L378